firebase-vscode/src/stubs/inquirer-stub.js

```diff
@@ -0,0 +1,24 @@
+const inquirer = module.exports;
+
+const optionsKey = Symbol('options');
+inquirer[optionsKey] = {};
+
+inquirer.setInquirerOptions = (inquirerOptions) => {
+  inquirer[optionsKey] = inquirerOptions;
+};
+
+inquirer.prompt = async (prompts) => {
+  const answers = {};
+  for (const prompt of prompts) {
+    if (inquirer[optionsKey].hasOwnProperty(prompt.name)) {
+      answers[prompt.name] = inquirer[optionsKey][prompt.name];
+    } else {
+      console.log(
```

Comment:
Should this be switched to use the logger you implemented?

Reply:
Did it in a somewhat roundabout way since the stubs are written in commonjs and can't import from ts files - could rewrite this in the future so the stubs are written in ts and compiled.